storage/client/blob/src/main/java/com/azure/storage/blob/BlobClientBuilder.java

```diff
@@ -152,6 +168,10 @@ public AppendBlobClient buildAppendBlobClient() {
     }
 
     /**
+     * Creates a {@link AppendBlobAsyncClient} based on options set in the Builder. AppendBlobAsyncClients are used to
+     * perform append blob specific actions such as {@link AppendBlobAsyncClient#appendBlock(Flux, long)}, only use this
```

Comment:
Looks like it should be {@link AppendBlobAsyncClient#appendBlock(Flux, long) append block} 

The "append block" at the end is missing